stores/identity.ts

```diff
@@ -0,0 +1,133 @@
+import { balanceOf, identityOf } from '@kodadot1/sub-api'
+import { Registration } from '@polkadot/types/interfaces/identity/types'
+import { defineStore } from 'pinia'
+import consola from 'consola'
+import { emptyObject } from '@/utils/empty'
+import { formatAddress } from '@/utils/account'
+
+export interface IdentityMap {
+  [address: string]: Registration
+}
+
+type BalanceMap = Record<string, string>
+type ChangeAddressRequest = {
+  address: string
+  apiUrl?: string
+}
+
+export interface Auth {
+  address: string
+  source?: 'keyring' | 'extension' | 'ledger'
+  balance?: BalanceMap
+  tokens?: BalanceMap // <id, amount>
+}
+
+export interface IdentityStruct {
+  identities: IdentityMap
+  auth: Auth
+}
+
+export interface IdenityRequest {
+  address: string
+  identity: Registration
+}
+
+export const useIdentityStore = defineStore('identity', {
+  state: (): IdentityStruct => ({
+    identities: emptyObject<IdentityMap>(),
+    auth: {
+      ...emptyObject<Auth>(),
+      balance: emptyObject<BalanceMap>(),
+      tokens: emptyObject<BalanceMap>(),
+      address: localStorage.getItem('kodaauth') || '',
```

Comment:
```useLocalStorage``` ?

Reply:
Looks like you helped me to discover that `useLocalStorage` isn't working as excepted.
`JSON.parse` throw error since `kodaauth` is not a JSON string, we need to serialize data beforehand or replace this composable with the one in VueUse